docs/windows.md

```diff
@@ -0,0 +1,35 @@
+# Windows Installation
```

Comment:
Can we document where the config file is installed and where to find logs?

Reply:
Its in the windows.md, for logs do you mean agent or the nssm?

Reply:
I'm having a hard time finding it, can you point it out to me? I see `CustomDirectory` but not the default directory where it is installed. 
  
>  for logs do you mean agent or the nssm?

Agent logs, depending on where NSSM is putting them